src/frontend/src/pages/sales/ReturnOrderDetail.tsx

```diff
@@ -120,15 +124,6 @@ export default function ReturnOrderDetail() {
         total: order.line_items,
         progress: order.completed_lines
       },
-      {
-        type: 'progressbar',
```

Comment:
Q: Is this replaced anywhere?

Reply:
No, return orders do not have shipments. This detail was an error, likely copied from the sales order detail page.